articles/ros_node_idl.md

```diff
@@ -85,7 +91,8 @@ Such assertions could include:
 These assertions results would then be summarized in a logging file for later debugging.
 
 Another example of the usefulness of having a static interface is the ability to create graphical tools for putting a ROS system together.
-Yet another example would be an additional feature in `ros2 pkg create` that would allow a developer to hand it an IDL and have it generate scaffolding for a node with that interface.
+
+Yet another example would be an additional feature in `ros2 pkg create` that would allow a developer to hand it an NoDL and have it generate scaffolding for a node with that interface.
```

Comment:
```suggestion
Yet another example would be an additional feature in `ros2 pkg create` that would allow a developer to hand it a NoDL and have it generate scaffolding for a node with that interface.
```